A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/EICAS/ECAM/A320_Neo_LowerECAM_Engine.js

```diff
@@ -5,7 +5,9 @@ var A320_Neo_LowerECAM_Engine;
     Definitions.MIN_GAUGE_OIL = 0;
     Definitions.MAX_GAUGE_OIL = 25;
     Definitions.MIN_GAUGE_PSI = 0;
+
     Definitions.MAX_GAUGE_PSI = 100;
+
```

Comment:
Remove spaces

Reply:
Tell that to ASOBO Dev Team, my friend, as it was like this in the original code !

Reply:
The diff shows that you added the lines.

If they are from Asobo, please try to clean up as much as possible anyway.